kroxylicious-kms-provider-aws-kms/DEV_GUIDE.md

```diff
@@ -0,0 +1,81 @@
+Here's some information that might be helpful when developing this module.
+
+# Testing the ability to authenticate with EC2 metadata
+
+In order to test this feature, you need to deploy Kroxylicious to an EC2 instance. The 
+EC2 instance needs to be associated with a properly configured IAM role. The EC2 credentials
+provider will use that to authenticate.
+
+Here's some instructions that will help you test end to end.  For simplicity, these instructions
+co-locate a Kafka Broker on the same instance.  Of course, you can choose to run the Kafka
+Cluster in any network accessible location.  These instructions also assume you want to run
+your Kafka client off-EC2.
+
+1. Create an EC2 instance on AWS - `t2.micro` instance will suffice.
+2. Enable SSH with key pair. The remainder of the instructions assume you've download the key into `.ssh/MyEC2KeyPair.pem`.
+3. Enable inbound traffic to port 22 (for SSH) and 9192 - 9193 (for Kafka traffic).
```

Comment:
I needed to expose 9194, I guess because the broker has id 1, maybe safest to expose all the ports Kroxy binds before discovery, so up to 9195